src/main/resources/idprovider/idprovider.xml

```diff
@@ -54,6 +54,13 @@
       <occurrences minimum="0" maximum="1"/>
     </input>
 
+    <input name="pageSize" type="Long">
+      <label>The page return size from graph api</label>
+      <default>5</default>
```

Comment:
Would using `5` as the default make the application work the same as before this PR?

Reply:
Different APIs might have different default, and maximum page sizes might behave differently if you specify a page size. Page size should therefore be optionally, I will change that part now.